CONTRIBUTING.md

```diff
@@ -255,6 +256,11 @@ This involves the following steps:
    will pull these formatting changes back into your local repository. To avoid merge conflicts
    it's a good habit to do this a few minutes after pushing anything to your remote repository.
 
+   Also remember that a push to a remote repository will usually spawn a number of Action-based self-tests. While these don't "cost" us anything (Thank you, Github!) you should still kill unnecessary runs (in the Actions tab) if the push to your forked repository is intended as a backup and/or has not affected parts of the library that are tested by these Actions. For instance, there's no need to re-run the self-tests if you've only worked on the documentation. You can cancel a particular workflow by clicking on the "..." button as shown here:
```

Comment:
> and/or has not affected parts of the library that are tested by these Actions. For instance, there's no need to re-run the self-tests if you've only worked on the documentation.

I would weakly disagree with this, I find that it's easy to convince myself that I definitely haven't broken something when I possibly have. It also takes mental load off reviewers to see all green before merging. Saying this, I agree with the general point though: if no one is ever going to care about the result, it's good to cancel it.

Reply:
Slightly rephrased to make it clear that a full set of self-tests should be run before submitting a pull request.